example_app_generator/generate_stuff.rb

```diff
@@ -167,4 +167,17 @@ def using_source_path(path)
 gsub_file 'spec/controllers/uploads_controller_spec.rb',
           'skip("Add a hash of attributes valid for your model")',
           '{}'
+
+# adds validation to widget so our test for happy and unhappy path will be ready to test validation
+gsub_file 'spec/requests/widgets_spec.rb',
+          'skip("Add a hash of attributes invalid for your model")',
+          '{ name: "" }'
+
+gsub_file 'spec/requests/widgets_spec.rb',
+          'skip("Add a hash of attributes valid for your model")',
+          '{ name: "lorem" }'
+
+gsub_file 'app/models/widget.rb',
+          'end',
+          'validates :name, presence: true; end'
```

Comment:
@pirj should I do it this way, or maybe I can just add validation to the widget model in the scaffold generator?

Reply:
It's quite good like this 👍